cmd/swapcli/main.go

```diff
@@ -379,12 +381,12 @@ func runBalances(ctx *cli.Context) error {
 	}
 
 	fmt.Printf("Ethereum address: %s\n", balances.EthAddress)
-	fmt.Printf("Balance: %s\n", common.FmtFloat((*common.WeiAmount)(balances.WeiBalance).AsEther()))
+	fmt.Printf("ETH Balance: %s\n", balances.WeiBalance.AsEther().Text('f'))
 	fmt.Println()
 	fmt.Printf("Monero address: %s\n", balances.MoneroAddress)
-	fmt.Printf("Balance: %s\n", common.FmtFloat(common.PiconeroAmount(balances.PiconeroBalance).AsMonero()))
-	fmt.Printf("Unlocked balance: %s\n",
-		common.FmtFloat(common.PiconeroAmount(balances.PiconeroUnlockedBalance).AsMonero()))
+	fmt.Printf("XMR Balance: %s\n", balances.PiconeroBalance.AsMonero().Text('f'))
```

Comment:
On `apd.Decimal` we have to use `Text('f')` instead of `String()` or it will sometimes use exponent syntax. With the `String()` method on our own types like `PiconeroAmount`, `WeiAmount` and `ExchangeRate`, it is not needed.